0344_ReverseString.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
```

Comment:
@Jovannna Here you can take a look at the pull request, before merge to master. You can look at the code, comment and ask for things to change.

Reply:
Napravio si test, super. Ja otkucala samo funkciju i razmisljam kako da unesem string i kako da ga stampam radi testa. A funkcija mi je ista.